packages/common/src/utils/validationUtils.ts

```diff
@@ -222,8 +239,23 @@ export const validateExpressions = (certificate: Certificate, updated: Certifica
       )
     }
   }
+  /**
+   * HIDE_VALIDATION has priority over SHOW_VALIDATION
+   */
+  function resolvePriorityBetweenValidationTypes(validationResult: ValidationResult) {
+    function hideValidationHasPriorityOverShow(validationResult: ValidationResult) {
+      return validationResults.some(
+        (value) => value.type === CertificateDataValidationType.HIDE_VALIDATION && value.id === validationResult.id
+      )
+    }
+
+    if (validationResult.type === CertificateDataValidationType.SHOW_VALIDATION) {
+      return !hideValidationHasPriorityOverShow(validationResult)
+    }
+    return true
+  }
 
-  return validationResults
+  return validationResults.filter((validationResult) => resolvePriorityBetweenValidationTypes(validationResult))
```

Comment:
Looks like this function is not being used, could be removed?

Reply:
The function is getting called in the filter on line 258.

Reply:
Oh, I thought that was from inside itself, but I see now. 👍